packages/synapse-constants/constants/tokens/bridgeable.ts

```diff
@@ -1263,3 +1268,22 @@ export const USDBC = new Token({
   routeSymbol: 'USDbC',
   coingeckoId: 'bridged-usd-coin-base',
 })
+
+export const USDB = new Token({
+  visibilityRank: 101,
+  addresses: {
+    [CHAINS.BLAST.id]: '0x4300000000000000000000000000000000000003',
+  },
+  decimals: {
+    [CHAINS.BLAST.id]: 18,
+  },
+  swapExceptions: {},
+  symbol: 'USDB',
+  name: 'Blast Rebasing USD',
+  logo: usdbLogo,
+  swapableType: 'USD',
+  swapableOn: [],
+  color: 'blue',
+  priorityRank: 125,
```

Comment:
`priorityRank` should be 1 here I think

Reply:
@Defi-Moses let's do this in next pr so we can hotfix explorer ui

Reply:
this won't go live to synapse-interface just yet anyway